src/silo/storage/dictionary.cpp

```diff
@@ -354,4 +354,11 @@ uint32_t Dictionary::getRegionCount() const {
    return region_count;
 }
 
+// NOLINTNEXTLINE(readability-convert-member-functions-to-static)
```

Comment:
Not important, but I think there is also //NOLINTBEGIN and //NOLINTEND

Reply:
This will resolve itself after a rebase on your branch anyway.